src/components/Incidents/IncidentsPage.jsx

```diff
@@ -176,6 +204,8 @@ const IncidentsPage = ({ customDataSource, namespace }) => {
             alertsData={alertsData}
             incidentsData={filteredData}
             chartDays={timeRanges.length}
+            chooseIncident={setChooseIncident}
+            tableLoaded={setAlertsAreLoading}
```

Comment:
if these are function props it might be more readable to use:
```suggestion
            onIncidentSelect={setChooseIncident}
            onTableLoaded={setAlertsAreLoading}
```